### PR TITLE
Push min Rust version back

### PR DESCRIPTION
**Pull request by CAD97**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: rust
 rust:
-- 1.22.0  # Two releases back
+- 1.16.0  # Min Rust Version
```
Comment (epage) [inline .travis.yml]: Any particular reason you are targeting 1.16?

Comment (CAD97) [reply .travis.yml]: `cargo check` 😆

Comment (CAD97) [reply .travis.yml]: It's going to bump up again, I went back to 1.15 which is where custom derive was introduced and am pushing it up by the errors.

I think the latest thing I used is `?`, which I forgot about until just now.

Comment (CAD97) [reply .travis.yml]: Basically, it's implemented now how I want it to be, and I'm trying to guarantee the largest range I can.

**Comment by CAD97** [comment]
How interesting, riding the errors I found out I was always requiring 1.22.0 as my minimum supported Rust version.

bors: r+

**Pull request merged**